odh/overlays/moc/kafka/kustomization.yaml

```diff
@@ -4,17 +4,18 @@ kind: Kustomization
 
 resources:
   - ../../../base/kafka
-  - topics
 
 patchesJson6902:
   - patch: |
```

Comment:
@HumairAK This patch only adds the topics overlay. Do we need something that also deploys the base? 
i.e. the [kafka cluster](https://github.com/operate-first/apps/blob/7f876c9a36e24c8c0502ecebbf6f2a7438a8d48b/odh/overlays/moc/kafka/overrides/kafka/base/kafka-cluster.yaml)?

Reply:
I think same would go for other environments as well

Reply:
nvm it deploys the base as well. Just tested :+1: